docs/source/release_notes/release-notes.rst

```diff
@@ -146,12 +173,6 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2003>`__)
 
 
-    *    - |devbreak| |improved|
-         - Converted all compile time uses of Guava's ``com.google.common.base.Optional`` class to the Java8 equivalent ``java.util.Optional``.
-           Should not affect deployments as there is no change to `json` or `yml` representations of AtlasDB configurations.
-           This is a relatively straightforward compile time break for products consuming AtlasDB libraries.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2016>`__)
-
```

Comment:
Where are the 0.44 release notes?  Also I think there are some notes missing:
https://github.com/palantir/atlasdb/pull/2014/files#diff-4d3c12695a1d39e530622f706c99ab37
https://github.com/palantir/atlasdb/pull/1891/files
(This only merged a few days ago but the changelog entry is in 0.40)